qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1182,6 +1182,19 @@ def remove_nondescendants_of(self, node):
             if n.type == "op":
                 self.remove_op_node(n)
 
+    def front_layer(self):
```

Comment:
Since this extends `DAGCircuit` API, maybe a test in `test/python//dagcircuit/test_dagcircuit.py`

Reply:
Test added in 5527c0512